backend/src/Jira/Service/JiraIssueService.php

```diff
@@ -42,7 +42,7 @@ public function loadIssue(ProjectAgentConnection $connection, string $jiraId): ?
         return new JiraIssue(
             $jiraId,
             $issue->fields->summary,
-            $issue->fields->description,
+            $issue->fields->description ?: '',
```

Comment:
It's important to ensure that the `description` field fallback to an empty string (`''`) if it's null or undefined. However, it seems like the code doesn't handle the case where the `description` field itself is missing in the `issue->fields` object, which could potentially lead to an undefined property access error.